tests/integration/security/authorization_test.go

```diff
@@ -1174,17 +1264,28 @@ func TestAuthorization_Path(t *testing.T) {
 						}
 					}
 					cases := []rbacUtil.TestCase{
-						newTestCase("/public", true),
-						newTestCase("/private", false),
-						newTestCase("/public/../private", false),
-						newTestCase("/public/./../private", false),
-						newTestCase("/public/.././private", false),
-						newTestCase("/public/%2E%2E/private", false),
-						newTestCase("/public/%2e%2e/private", false),
-						newTestCase("/public/%2E/%2E%2E/private", false),
-						newTestCase("/public/%2e/%2e%2e/private", false),
-						newTestCase("/public/%2E%2E/%2E/private", false),
-						newTestCase("/public/%2e%2e/%2e/private", false),
+						newTestCase(a, "/public", true),
```

Comment:
here and other places, please make this a loop and reuse the test cases instead of duplicating all the test cases.